docs/components/typography/test/index.html

```diff
@@ -0,0 +1,299 @@
+{% extends 'test.njk' %} {% block content %}
+<section>
+    <!-- Caption -->
+    <div class="demo" data-dom-diff="typography-caption">
+        <p class="hxCaption">Caption</p>
+    </div>
+</section>
+<section>
+    <!-- Error Message -->
+    <div class="demo" data-dom-diff="typography-error-message">
+        <hx-error>Error message goes here</hx-error>
+    </div>
+</section>
+<section>
+    <!-- Inline Code -->
+    <div class="demo" data-dom-diff="typography-inline-code">
+        <p>
+            Add an event listener with the
+            <code>EventTarget.addEventListener()</code> function.
+        </p>
+    </div>
+</section>
+<section>
+    <!-- Keyboard -->
+    <div class="demo" data-dom-diff="typography-keyboard">
+        <p>
+            Press
+            <kbd>Ctrl+Z</kbd> (
+            <kbd>Command+Z</kbd> on Mac) to undo the most recent action.
+        </p>
+    </div>
+</section>
+<section>
+    <!-- Links -->
+    <div class="hxAlert">
+        <div class="hxAlert__icon">
+            <hx-icon type="info-circle"></hx-icon>
+        </div>
+        <div class="hxAlert__text">
+            <span class="hxAlert__status">Note</span>
+            Buttons are better suited to initiate in-page actions.
+        </div>
+    </div>
+    <div class="demo" data-dom-diff="typography-link">
+        <ul>
+            <li>
+                <a href="#">Normal Link</a>
+            </li>
+            <li>
+                <a href="#" class="pseudo-active">Pressed Link</a>
+            </li>
+        </ul>
+    </div>
+</section>
+<section>
```

Comment:
Remove section. There is no Shadow DOM that needs testing.

Reply:
removed